db/schema.rb

```diff
@@ -27,4 +27,12 @@
     t.index ["reset_password_token"], name: "index_admins_on_reset_password_token", unique: true
   end
 
+  create_table "warehouses", force: :cascade do |t|
+    t.integer "code"
+    t.string "name"
+    t.string "location"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
```

Comment:
I'd recommend to add something to perform logical deletes, otherwise, if you delete a warehouse all historical data would be lost, at least, a state column to see if the warehouse is active or not

Reply:
@pacheco-andres solo te queda este pendiente